src/schema/v2/artwork/index.ts

```diff
@@ -141,10 +147,17 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
               "Use whatever is in the original response instead of making a request",
           },
         },
-        resolve: ({ artists }, { shallow }, { artistLoader }) => {
+        resolve: (
+          { artists },
+          { shallow },
+          { authenticatedArtistLoader, artistLoader }
+        ) => {
           if (shallow) return artists
+
+          const loader = authenticatedArtistLoader || artistLoader
```

Comment:
We need an authenticated request to Gravity in case we fetch a personal artist (artists created by the user). In the case the request to MP is not authenticated, `authenticatedArtistLoader` is undefined and we fall back to the unauthenticated `artistLoader`.